doc/user/content/sql/show-types.md

```diff
@@ -6,16 +6,18 @@ menu:
     parent: commands
 ---
 
-`SHOW TYPES` returns a list of the data types in Materialize. By default, only custom types are returned.
+`SHOW TYPES` returns a list of the data types in Materialize. Only custom types
```

Comment:
Since it didn't seem like you could do non-default behavior. Unless our page in incomplete (which it may well be).